directories/README.md

```diff
@@ -26,6 +26,13 @@ The library provides the location of these directories by leveraging the mechani
 This library is written in Rust, and supports Linux, Redox, macOS and Windows.
 Other platforms are also supported; they use the Linux conventions.
 
+## Minimum Rust version policy
```

Comment:
Could you rebase against upstream master branch? This diff looks weird.

Reply:
I'm a bit new to all this but I think I did what you asked for :) Could you check now?